.circleci/config.yml

```diff
@@ -43,13 +43,21 @@ jobs:
       - attach_workspace:
           at: ~/
       - run:
-          name: Validating code formatting
+          name: Run fmt-test
           command: make fmt-test
       - run:
-          name: Vetting code
+          name: Installing golangci-lint
+          command:  go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.26.0
+          environment:
+            GO111MODULE=on
+      - run:
+          name: Run lint
+          command: make lint
+      - run:
+          name: Run vet
```

Comment:
@sameersbn one note about this: `vet` is already part of the linters [enabled by default](https://github.com/golangci/golangci-lint#enabled-by-default-linters) in `golandci-lint`. Do you mind if I remove this?

Reply:
sounds good to me :+1: